client/src/main/scala/drt/client/components/MultiDayExportComponent.scala

```diff
@@ -66,15 +66,14 @@ object MultiDayExportComponent extends WithScalaCssImplicits {
       }
 
       val gridXs = 4
-      <.div(
         MuiButton(color = Color.default, variant = "outlined", size = "medium")(
           MuiIcons(GetApp)(fontSize = "small"),
           "Multi Day Export",
           ^.className := "btn btn-default",
           VdomAttr("data-toggle") := "modal",
           VdomAttr("data-target") := "#multi-day-export",
           ^.onClick --> scope.modState(_.copy(showDialogue = true))
-        ),
+        )
```

Comment:
The comma needs to be restored here, and the div needs to be reinstated around these two components